### DISCUSSION TITLE
Please make new release 0.4.0-alpha.5

### DISCUSSION THREAD

**Discussion by KaKi87** [florisboard/florisboard#2154] [Q&A]
Hello,

Please make a new release in order to enjoy the latest bugfixes.

Thanks

**Comment by KaKi87** [comment]
Up

**Comment by franco27** [comment]
I too wait impatiently for a new version.

 I really like this keyboard. I hope the project is not abandoned.

 I am French. And I'm waiting for the functionality of words suggestions.

 I guess the development of this function is difficult to implement, with all languages and requires more time ... Yes, the creation of new versions has interrupted.

**Comment by patrickgold** [comment]
Please see https://github.com/florisboard/florisboard/discussions/2197

**Reply by KaKi87** [reply]
Thanks but my request was about getting already completed bug fixes, not features in progress.

**Reply by patrickgold** [reply]
The problem is the current main branch contains some bug fixes but also unresolved bugs which I need to address before releasing, so they will get released together with the NLP release.

**Comment by franco27** [comment]
I thank you for all these details.
